ocw/lib/vault.py

```diff
@@ -24,7 +24,11 @@ def __init__(self, vault_namespace):
         self.namespace = vault_namespace
         self.password = cfg.get(['vault', 'password'])
         self.certificate_dir = cfg.get(['vault', 'cert_dir'], '/etc/ssl/certs')
-        self.auth_json = None
+        if cfg.getBoolean(['vault', 'use-file-cache']) and self._getAuthCacheFile().exists():
+            logger.info('Loading cached credentials')
+            self.auth_json = self.loadAuthCache()
+        else:
+            self.auth_json = None
```

Comment:
This doesn't look related to this PR

Reply:
this is why it is a separate **commit** , do you also want to create a separate github PR for it ?

Reply:
no, sorry didn't realized